rules/promotions/external_alerts.toml

```
@@ -19,7 +19,7 @@ risk_score = 47
 rule_id = "eb079c62-4481-4d6e-9643-3ca499df7aaa"
 rule_name_override = "message"
 severity = "medium"
-tags = ["Elastic", "Windows", "APM", "Network", "macOS", "Linux"]
+tags = ["Elastic", "Network", "Windows", "APM", "macOS", "Linux"]
```

```suggestion
tags = ["Elastic", "Promotion"]
```

Need some additional feedback on this one.  I don't like the current form either way.

---

i'd drop promotion too.
just `["Elastic"]` is fine imho

---

```suggestion
tags = ["Elastic"]
```

---

This one is breaking testing.  @paulewing, can @brokensound77 and I get a ruling?